### PR TITLE
🗞️ Adding NativeOFTAdapter example to devtools

### PR DESCRIPTION
**Pull request by ravinagill15**
## In this PR
- Added NativeOFTAdapter example to devtools

### CODE REVIEW

**Comment by ravinagill15** [comment]
Please do not merge until the `NativeOFTAdapter` contract within `feat/native-oft-adapter` has been audited

File: tests-user/tests/create-lz-oapp.bats [hunk 1]
```diff
@@ -211,6 +222,17 @@ teardown() {
     yarn lint:fix
 }
 
+@test "should work with yarn & native-oft-adapter example in CI mode" {
+    local DESTINATION="$PROJECTS_DIRECTORY/yarn-oapp"
```
Comment (ravinagill15) [inline tests-user/tests/create-lz-oapp.bats:226]: Should this be `yarn-native-oft-adapter` instead? I copied this over from the `oft-adapter` test right above which also lists the destination as `yarn-oapp`. I can fix both if this is a typo

File: tests-user/tests/create-lz-oapp.bats [hunk 2]
```diff
@@ -264,6 +286,17 @@ teardown() {
     npm run lint:fix
 }
 
+@test "should work with npm & native-oft-adapter example in CI mode" {
+    local DESTINATION="$PROJECTS_DIRECTORY/npm-oft"
```
Comment (ravinagill15) [inline tests-user/tests/create-lz-oapp.bats:290]: Same question as above here -- Should this be `npm-native-oft-adapter` instead? I copied this over from the oft-adapter test right above which also lists the destination as `npm-oft`. I can fix both if this is a typo

File: examples/native-oft-adapter/deploy/MyNativeOFTAdapter.ts [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    //     ...
+    //     eid: EndpointId.AVALANCHE_V2_TESTNET
+    //   }
+    // }
+    const endpointV2Deployment = await hre.deployments.get('EndpointV2')
+
+    const { address } = await deploy(contractName, {
+        from: deployer,
+        args: [
+            18, // TODO update this with native local decimals
```
Comment (ryandgoulding) [inline examples/native-oft-adapter/deploy/MyNativeOFTAdapter.ts:39]: This is where I really dislike deploy scripts, as they do not support mandatory arguments.  `18` is almost always correct, unless it isn't, which is rare but occasionally happens.  EVM has no API to determine gas token decimals that I am aware of (you would likely need to call a known WETH9 precompile).  I think this is the best we can do given the limitations, thoughts?

Comment (ravinagill15) [reply examples/native-oft-adapter/deploy/MyNativeOFTAdapter.ts:39]: Thanks for the feedback! I agree that hardcoding 18 isn't ideal, but given the lack of an API to determine gas token decimals, this _seems_ like the best option for now. Open to revisiting this if we find a better solution!

**Review by ryandgoulding — DISMISSED** [review]

**Review by ryandgoulding — APPROVED** [review]

**Pull request merged**